app/services/account/account_email/account_email.go

```diff
@@ -45,7 +64,58 @@ func (m *Manager) Add(ctx context.Context, accountID account.AccountID, address
 }
 
 func (m *Manager) Remove(ctx context.Context, accountID account.AccountID, id xid.ID) error {
-	err := m.emailRepo.Remove(ctx, accountID, id)
+	// Get account with all email addresses and auth methods
+	acc, err := m.accountQuerier.GetByID(ctx, accountID)
+	if err != nil {
+		return fault.Wrap(err, fctx.With(ctx))
+	}
+
+	// Get all authentication methods for this account
+	authMethods, err := m.authRepo.GetAuthMethods(ctx, accountID)
+	if err != nil {
+		return fault.Wrap(err, fctx.With(ctx))
+	}
+
+	// Check if account has OAuth authentication (Discord or GitHub)
+	hasOAuthAuth := lo.ContainsBy(authMethods, func(auth *authentication_repo.Authentication) bool {
+		return auth.Service == authentication_repo.ServiceOAuthDiscord ||
+			auth.Service == authentication_repo.ServiceOAuthGitHub ||
+			auth.Service == authentication_repo.ServiceOAuthGoogle
+	})
+
+	// Count verified emails
+	verifiedEmails := lo.Filter(acc.EmailAddresses, func(e *account.EmailAddress, _ int) bool {
+		return e.Verified
+	})
+
+	// Find the email being deleted
+	emailToDelete, found := lo.Find(acc.EmailAddresses, func(e *account.EmailAddress) bool {
+		return xid.ID(e.ID) == id
+	})
+
+	if !found {
+		return fault.New("email address not found",
+			fctx.With(ctx),
+			ftag.With(ftag.NotFound),
+		)
+	}
+
+	// If trying to delete a verified email and it's the last verified one
+	if emailToDelete.Verified && len(verifiedEmails) == 1 {
+		// Check if there are OAuth connections
+		if !hasOAuthAuth {
+			return fault.New("cannot delete last verified email address",
+				fctx.With(ctx),
+				ftag.With(ftag.PermissionDenied),
+				fmsg.WithDesc(
+					"last verified email",
+					"You cannot delete your last verified email address without linking an OAuth account (Discord, GitHub, or Google) first. Please add an OAuth authentication method before removing this email.",
```

Comment:
Since OAuth providers are 1. dynamic (they may be enabled/disabled) and 2. a growing list over time (we'll add more) it's best not to mention specific auth methods in the error, instead use more generic language like:

> You cannot delete your only verified email. Please add another verified email address first.